src/styles/navigation.less

```diff
@@ -88,6 +88,16 @@
                 color: #fff;
             }
         }
+
+        .parent {
+            margin-top: 6px;
```

Comment:
@brendanmig this CSS change messes visually with the list of properties in a way that I'd rather not change. Can you please tweak to be only specific to tutorials?

## Before
![Screen Shot 2019-05-29 at 2 14 01 PM](https://user-images.githubusercontent.com/864393/58591844-0e2c7300-821c-11e9-8127-460923e0db42.png)

## After
![Screen Shot 2019-05-29 at 2 14 07 PM](https://user-images.githubusercontent.com/864393/58591855-14225400-821c-11e9-88b9-117e4a184303.png)

Reply:
```suggestion
-            margin-top: 6px;
```
This actually isn't needed at all. The tutorials use a `margin-bottom` for the spacing and this `margin-top` isn't doing anything other than causing problems elsewhere.